minecraft_servers/playvenify/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+    "server_name": "playvenify",
+    "nice_name": "PlayVenify",
+    "direct_ip": "mc.playvenify.net",
+    "server_wildcards": [
+      "%.playvenify.net",
+      "%.nurserver.net"
+    ],
+    "supported_languages": [
+      "de", 
+      "en"
+    ],
+    "social": {
+      "web": "https://www.playvenify.net",

```

Comment:
We only accept pages that are online.

Reply:
I have removed the website from the "manifest.json".